pageobjects/login.page.js

```diff
@@ -0,0 +1,30 @@
+// login.page.js
+
+class LoginPage {
+
+    get usernameInput () { return $('#username') }
+    get passwordInput () { return $('#password') }
+    get submitBtn () { return $('#submit') }
+    
+    //Enter "login" page
+    async openWeb() {
+        await browser.url('login');
+        //await browser.url('login');
+    }
+
+    //Press submit button
+    async logIn (username, password) {
+
+        await this.usernameInput.waitForClickable();
+        await this.usernameInput.setValue(username);
```

Comment:
There is duplication of the pattern of waiting for an element to be clickable and setting their value here. Consider encapsulating that logic into its own method: Something like `completeField(element, text)` that waits for an element to be clickable and then sends it the string you pass it by parameter. That way you reduce overall lines for `logIn()` method and make it more readable (as you make it work with constructs of higher-level of abstraction).